hedgehog-example/test/Test/Example/Basic.hs

```diff
@@ -251,6 +254,24 @@ prop_record =
     x === y
 
 ------------------------------------------------------------------------
+-- Example 6 - Text.takeEnd
+
+prop_takeEnd :: Property
+prop_takeEnd =
+  property $ do
+    xs <- forAll $ Gen.string (Range.linear 0 100) Gen.unicode
```

Comment:
If this is changed from `Gen.unicode` to `Gen.latin1` then the test will start passing